kubejs/server_scripts/mods/modern_industrialization/blast_furnace.js

```diff
@@ -126,7 +137,8 @@ ServerEvents.recipes(e => {
         512,
         1200,
         [ { amount: 1, tag: 'c:superconductor_dusts' } ],
-        [ { amount: 1, item: mi('superconductor_hot_ingot') } ]
+        [ { amount: 1, item: mi('superconductor_hot_ingot') } ],
+        [ { amount: 250, fluid: mi('molten_nether_star') } ]
```

Comment:
250 seems like an extremely high amount. Even with a fully overclocked Mob Crusher, I feel like players would be waiting on the 2.5% chance for Wither Skeleton Skulls to just get two superconductor ingots out of it.

Reply:
I've bumped it down to 50, can go down to 25 or 15 if you think 50 is too much.